app/assets/sass/application.scss

```diff
@@ -12,16 +12,21 @@ $path: "/public/images/";
 button.link {
   color: #005ea5;
   text-decoration: underline;
-  font-size:18px;
+  font-size:16px;
```

Comment:
in general we should not be using fixed px sizes for fonts, using em instead, but will leave to CSS clean up issue